test_aodndata/soop/test_soop_co2.py

```diff
@@ -10,9 +10,7 @@
 GOOD_NC = os.path.join(TEST_ROOT, 'IMOS_SOOP-CO2_GST_20170126T023510Z_VNAA_FV01.nc')
 GOOD_FRMAP = os.path.join(TEST_ROOT, 'FutureReefMap_GST_20150518T124011Z_9V2768_FV01.nc')
 GOOD_ZIP = os.path.join(TEST_ROOT, 'IMOS_SOOP-CO2_GST_20170126T023510Z_VNAA_FV01.zip')
-GOOD_RT_IN_TXT = os.path.join(TEST_ROOT, 'IN_2018-022-0000dat.txt')
-GOOD_RT_IN_2_TXT = os.path.join(TEST_ROOT, 'IN_2019-345-0000dat.txt')
-GOOD_RT_AA_TXT = os.path.join(TEST_ROOT, 'AA_2019-032-0001dat.txt')
+GOOD_RT_IN_TXT = os.path.join(TEST_ROOT, 'IN_2022-332-0000dat.txt')
 
 ship_callsign_ls = {'VNAA': 'Aurora-Australis',
```

Comment:
Do you still need to test files for the Aurora Australis?

Reply:
I need to keep testing  Aurora files in delayed mode only